src/main/java/de/zalando/aruha/nakadi/controller/EventPublishingController.java

```diff
@@ -75,6 +81,17 @@ private ResponseEntity postEventInternal(String eventTypeName, String eventsAsSt
         }
     }
 
+    private ResponseEntity processJSONException(final JSONException e, final NativeWebRequest nativeWebRequest) {
+        if (e.getCause() == null) {
```

Comment:
@adyach do you think this is relevant to our discussion http://zalando.github.io/restful-api-guidelines/common-data-objects/CommonDataObjects.html#must-an-error-message-must-not-contain-the-stack-trace ?